model/postgres/postgres_server.rb

```diff
@@ -88,7 +88,7 @@ def configure_hash
         }
       },
       identity: resource.identity,
-      hosts: "#{resource.representative_server.vm.ephemeral_net4} #{resource.identity}"
```

Comment:
What happens if we reconfigure existing resources? They're not on the same subnet

Reply:
What do you mean? PrivateSubnets span the whole region. We always put them into the same subnet.

Reply:
Weren't the primary and standby in different subnets before this PR? So primary and standy aren't in the same subnet for the existing PG resources.

Reply:
Oh, now I understand. Yes, this is a problem after migration for old resources.

Reply:
There are only 2 such resources. I can migrate them after merging this PR.

Reply:
The migration would involve creating new standby instances and deprovisioning the previous one, I suppose? Just wanted to make sure.

Reply:
Yep. Actually just deprovisioning old one. New one would get created automatically. There would be no customer impact.